### ISSUE TITLE
Files blanking, but not impacted on actual site (thank goodness)

### ISSUE THREAD

**Issue by awilliams84** [humy2833/FTP-Simple#273]
Issue Type: <b>Bug</b>

When I open a file, it's full of code. Then I click to another file and then come back to the original file, it's empty.

Any idea what's going on? TIA

Extension version: 0.7.5
VS Code version: Code 1.54.1 (f30a9b73e8ffc278e71575118b6bf568f04587c8, 2021-03-04T22:38:31.419Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4300U CPU @ 1.90GHz (4 x 2494)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.91GB (3.33GB free)|
|Process Argv|--crash-reporter-id 49b70fb1-590b-4ad2-84de-ef6ccf4a9c2c|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30256636
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
dockersubset:30265998
pythontbcf:30265426
vspre833cf:30267465

```

</details>

**Comment by nowayoutPL** [comment]
Same problem here. Files loads empty from server. Revert back to 1.53.2 temporarily fixes the problem.

**Comment by cloudfordream** [comment]
> Same problem here. Files loads empty from server. Revert back to 1.53.2 temporarily fixes the problem.

I have same problem i can't find version 1.53.2. Do you have more informations ?

**Comment by awilliams84** [comment]
@nowayoutPL you just saved me additional headache. I was trying to figure out if it was my host or FTP issue. Thanks!!

@cloudfordream https://code.visualstudio.com/updates/v1_53 links towards the top for download. I just did it and looks like it works great again. I also disabled updates for now.

**Comment by cloudfordream** [comment]
> @nowayoutPL you just saved me additional headache. I was trying to figure out if it was my host or FTP issue. Thanks!!
> 
> @cloudfordream https://code.visualstudio.com/updates/v1_53 links towards the top for download. I just did it and looks like it works great again. I also disabled updates for now.

Thanks for the link, same thing i disabled updates.

**Comment by humy2833** [comment]
The vscode has been upgraded and some features seem to have been removed.
Fixed in version 0.7.6(__)
